docs/samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-pg-v4-Y-Z-v4-full-DU.yaml

```diff
@@ -0,0 +1,376 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: sno-ran-du-pg-v4-Y-Z-v4
+policyDefaults:
+  namespace: ztp-sno-ran-du-v4-Y-Z
+  # Use an existing placement rule so that placement bindings can be consolidated
+  placement:
+    # These labels must match the labels set for the ManagedCluster either through the ProvisioningRequest
+    # or the ClusterInstance ConfigMap.
+    labelSelector:
+      cluster-version: "v4-Y-Z"
+      sno-ran-du-policy: "v4"
+  remediationAction: enforce
+  severity: low
+  namespaceSelector:
+    exclude:
+      - kube-*
+    include:
+      - '*'
+  evaluationInterval:
+    compliant: 5m
+    noncompliant: 10s
+  orderPolicies: true
+policies:
+# REDUCE FOOTPRINT
+- name: v4-footprint-policy
+  manifests:
+    # Do not add retention field for prometheusK8s when observability is enabled.
+    # It will be overridden by OBS.
+    - path: source-crs/ReduceMonitoringFootprint.yaml
+      patches:
+      - data:
+          config.yaml: |
+            alertmanagerMain:
+              enabled: false
+            telemeterClient:
+              enabled: false
```

Comment:
By default, observability is not enabled. I think we should just use the default values from source-cr.

Reply:
we should enable observability.